site/layouts/partials/community_links.html

```diff
@@ -10,11 +10,16 @@ <h2>Learn and Connect</h2>
 </div>
 <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
 <h2>Develop and Contribute</h2>
+<p>minikube is a Kubernetes <a href="https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle">#sig-cluster-lifecycle</a> project.</p>
 <p>If you want to get more involved by contributing to {{ .Site.Title }}, join us here:
 {{ with index $links "developer"}}
 {{ template "community-links-list"  . }}
 {{ end }}
-<p>You can find out how to contribute to these docs in our <a href="/contributing/guide">Contributing Guide</a>.
+<p>You can find out how to contribute to these docs in our <a href="/docs/contribution-guidelines/">Contribution Guidelines</a>.
+<p>Join our meetings: </p>
```

Comment:
make this as big as <h2>Develop and Contribute</h2>

Reply:
Done.